### ISSUE TITLE
[api_2] Batch operations?

### ISSUE THREAD

**Issue by martinrotter** [wallabag/wallabag#1922]
Will APIv2 support batch operations? Marking several entries archived etc.? Can API support marking of entries read/unread?

**Comment by j0k3r** [comment]
There are no batch operations at the moment. You have to do it one by one.
Could be an interesting feature.

**Comment by gnppn** [comment]
I agree it could be useful. With 500+ unread entries in my wallabag, sorting my articles with a refresh of the page for each operation (on the web version) can be a chore.

**Comment by Suncatcher** [comment]
> Could be an interesting feature.
I'd rather call it not *could be* but must have feature. I also have 500+ entries and I cannot process them one by one. 
Any updates about when this can be implemented?

**Comment by gnppn** [comment]
Hey!

I now have more than 2.000 unread articles in my wallabag.it account and this is really becoming unmanageable. Do you think batch operations to archive/delete entries could be coming in the next months?

Thanks!

**Comment by nicosomb** [comment]
Hello @gnppn. 

Batch operations feature is not yet planned, because we need to work on 2.3 (which will be released soon ... I hope!). But then, we possibly work on this feature.

**Comment by Suncatcher** [comment]
Very frustrating. It's a basic feature to implement but very appreciated.

**Comment by j0k3r** [comment]
We do know this is an appreciated feature. Even if it's a basic one from the user POV it's a complicated one to implement from a back POV but also from a front POV.
If we fail to properly implement the frontend part the feature will be useless.

Also, since we don't have enough time to work on all _basic features_ requested by users we focus on finishing what we started before working on a new one.

**Comment by gnppn** [comment]
Is it a realistic feature in the coming months, now that the 2.3 version is (almost) out? :)

Thanks for your work!

**Comment by j0k3r** [comment]
Nobody says it'll be in the 2.3.2.
It's pity that we aren't 50 developers to work on wallabag to be able to ship all requested features :roll_eyes: 
But be sure it's a really interesting feature (even it's a bit complex to implement)

**Comment by ewoks** [comment]
I said 2.3.2 because that's the current version on app.wallabag.it ... I bet it is interesting and useful feature and I am more than sure everything would be easier if 50 instead of 5-8 people are contributing regularly. I want to emphasise that users are thankful to your free time invested in this project

There are few questions about development process that I am not sure where to find answers:
- When are planned releases? When is next release?
- What is planned for which release? (link to trello or similar)
- When release will be deployed to app.wallabag.it? 
- Who & how is making decisions in project? 

I guess clear answers to these questions (similar to very good example of [release process](https://github.com/wallabag/wallabag/blob/master/RELEASE_PROCESS.md)), available as basic description in CONTRIBUTING.md would attract more people.

Nobody wish to invest time in something that will be shipped in 1 year or later (maybe) or process is not transparent or known so time can be wasted... With clear plans and fixed release cycles one could know what to expect
